src/validators/decimal.js

```diff
@@ -0,0 +1,2 @@
+import { regex } from './common'
+export default regex('numeric', /^-?[0-9.]*$/)

```

Comment:
I think that it is necessary to change 'numeric' to 'decimal'.

I think that there is a bug, as below.

```js
/^-?[0-9.]*$/.test('..')
// true
```

Reply:
Thanks for the feedback. I updated the regex to disallow `..`, `123.4.5`, and `123.`